lib/spack/spack/container/images.json

```diff
@@ -1,36 +1,86 @@
 {
   "images": {
-    "ubuntu:18.04": {
-      "os_package_manager": "apt",
-      "build": "spack/ubuntu-bionic",
-      "build_tags": {
-        "develop": "latest"
-      }
+    "alpine:3": {
+      "bootstrap": {
+        "template": "container/alpine_3.dockerfile"
+      },
+      "os_package_manager": "apk"
```

Comment:
This is a larger topic, but is it worth storing the list of OS packages here instead of in the templates?  One kind of interesting idea is that we could have a command for Spack to install them itself, using the OS package manager, at least until Spack knows how to bootstrap itself using only builtin python capabilities (which I'd like to eventually enable)

This would also decouple things from the OS and reduce the number of templates needed -- you could allow a user to specify any base image, even if it doesn't match the ones here, so long as they provided the necessary dependencies themselves.  But, if Spack detects an OS it knows how to bootstrap, it could install what it needs.

Reply:
I like this idea. Would it be fine with you to work on it in a separate PR?